tar/chunks/chunks_write.c

```diff
@@ -13,11 +13,14 @@
 
 #include "chunks_internal.h"
 #include "hexify.h"
+#include "tarsnap_glob.h"
 #include "rwhashtab.h"
 #include "warnp.h"
 
 #include "chunks.h"
 
+uint64_t tarsnap_glob_newbytes = 0;
```

Comment:
Rather than having a global variable, can we pass the chunks cookie to the siginfo code so that it can pull the data out of the chunks layer statistics which we're already keeping?

Reply:
The issue is how to link siginfo code and the chunks layer.

- siginfo is all part of `bsdtar.h`.
- `CHUNKS_W` is owned by the multitape layer.

I've sketched a version of what I think you're suggesting, but it has a few WTF moments.  I kind-of hope that I misunderstood something.

- `chunks_write_newzbytes()` refers to chunkstats `.s_zlen` directly (where previously that was only used in `chunks_stats_internal.c` and `chunks_directory.c`.  [I could fix this by adding a `chunks_stats_calcznew()` function, but that seemed like overkill at this stage.]

- `multitape_write.c` now includes `bsdtar.h`, for access to the new `siginfo_setchunks()`.

- `siginfo.c` now includes `chunks.h`.

- `siginfo.c` now contains a `static CHUNKS_W * chunks_write_cookie`.

It might be cleaner to put `CHUNKS_W` in `struct bsdtar` directly?